test/extended/image_ecosystem/plugin.go

```diff
@@ -770,6 +771,65 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 
 		})
 
+		g.It("jenkins-plugin test imagestream SCM", func() {
+			jobName := "test-imagestream-scm"
+			g.By("creating a jenkins job with an imagestream SCM")
+			data := j.readJenkinsJob("imagestream-scm-job.xml", oc.Namespace())
+			j.createItem(jobName, data)
+
+			// Because polling is enabled, a job should start automatically and fail
+			// Wait long enough for nothing to be running
+			time.Sleep(3 * time.Minute)
+
+			// Now create a new imagestream tag and expect a job to be kicked off
+			// that will create a new tag in the current namespace
+			g.By("creating an imagestream tag in the current project")
+			oc.Run("tag").Args("openshift/jenkins:latest", fmt.Sprintf("%s/testimage:v1", oc.Namespace())).Execute()
+
+			// Now wait after the image has been tagged for the Jenkins job to run
+			// and create the new imagestream/tag
+			g.By("verifying that the job ran by looking for the resulting imagestream tag")
+			err := wait.Poll(5*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
You might consider using framework.go:WaitForAnImageStreamTag to keep things DRY. Until #11570 merges, it only waits 1 minute, but you can bump it to 10 and I will rebase #11570 if you merge first.